packages/xchain-radix/src/client.ts

```diff
@@ -0,0 +1,482 @@
+   * @param intent_hash - The transaction intent hash
+   * @returns a transaction in Tx type
+   */
+  async convertTransactionFromHex(transaction_hex: string, intent_hash: string, confirmed_at: Date): Promise<Tx> {
+    const binaryString = Buffer.from(transaction_hex, 'hex').toString('binary')
+    const transactionBinary = new Uint8Array(binaryString.split('').map((char) => char.charCodeAt(0)))
+    const transactionSummary = await LTSRadixEngineToolkit.Transaction.summarizeTransaction(transactionBinary)
+    const withdrawAccount = Object.keys(transactionSummary.withdraws)[0]
+    const depositAccount = Object.keys(transactionSummary.deposits)[0]
+    const withdrawResource = Object.keys(transactionSummary.withdraws[withdrawAccount])[0]
+    const withdrawAmount: number = transactionSummary.withdraws[withdrawAccount][withdrawResource].toNumber()
+
+    const transaction: Tx = {
+      from: [
+        {
+          from: withdrawAccount,
+          amount: baseAmount(withdrawAmount),
+          asset: { symbol: withdrawResource, ticker: withdrawResource, synth: false, chain: 'radix' },
+        },
+      ],
+      to: [
+        {
+          to: depositAccount,
+          amount: baseAmount(withdrawAmount),
+          asset: { symbol: withdrawResource, ticker: withdrawResource, synth: false, chain: 'radix' },
+        },
+      ],
+      date: confirmed_at,
+      type: TxType.Transfer,
+      hash: intent_hash,
+      asset: { symbol: withdrawResource, ticker: withdrawResource, synth: false, chain: 'radix' },
+    }
+    return transaction
+  }
+
+  /**
+   * Creates a transaction using the SimpleTransactionBuilder, signs it with the
+   * private key and returns the signed hex
+   * @param params - The transactions params
+   * @returns A signed transaction hex
+   */
+  async transfer(params: TxParams): Promise<string> {
+    const networkId = this.network === Network.Mainnet ? NetworkId.Mainnet : NetworkId.Stokenet
+    const transactionConstructionUrl = `${this.getGatewayUrl()}${TRANSACTION_CONSTRUCTION_PATH}`
+    const fromAccount = await this.getAddressAsync()
+    try {
+      const constructionMetadataResponse = await axios.post(transactionConstructionUrl)
+      const builder = await SimpleTransactionBuilder.new({
+        networkId: networkId,
+        validFromEpoch: constructionMetadataResponse.data.ledger_state.epoch,
+        fromAccount: fromAccount,
+        signerPublicKey: this.privateKey.publicKey(),
+      })
+
+      const transaction = await builder
+        .transferFungible({
+          toAccount: JSON.parse((await this.prepareTx(params)).rawUnsignedTx)['toAccount'],
+          resourceAddress: JSON.parse((await this.prepareTx(params)).rawUnsignedTx)['resourceAddress'],
+          amount: JSON.parse((await this.prepareTx(params)).rawUnsignedTx)['amount'],
+        })
+        .compileIntent()
+        .compileNotarizedAsync(async (hash) => this.privateKey.signToSignature(hash))
+      return transaction.toHex()
```

Comment:
1. got it
2. nope. transfer for this method name is a little bit missleading. we do the actual transfer with the broadcastTx method.

Reply:
@0xOmarA on 1, I actually need to return the transaction hex so that the client can later on use it to broadcast the transaction...aka send it to the gateway

Reply:
Based on the other implementations, I see that the `transfer` function should construct the transaction and submit it to the network. This is what I see in the examples that they provide here: https://docs.xchainjs.org/xchain-client/xchain-ethereum/how-to-use.html#transfer-ethereum-using-ethereum-client

Reply:
yes the current implementation is actually sending the transaction